### ISSUE TITLE
Inactive window after launching Frescobaldi 3.3.0 on MacMini M1 - OS 13.2.1

### ISSUE THREAD

**Issue by TonioMus** [frescobaldi/frescobaldi#1585]
–> ARM Frescobaldi new code # 3.3.0 / OS 13.2.1
In fact, the main window appears in front of all the others after clicking again on the app icon on the doc (see #428), BUT
1/ no change in the name of the foreground app (ie reading "Safari" on my screen shot)
2/ consequently, no actions possible inside the app window
3/ "Forcer à quitter" proposed in the doc, instead of "quitter"…
[Inactive foreground windows.pdf](https://github.com/frescobaldi/frescobaldi/files/11171393/Inactive.foreground.windows.pdf)

**Comment by ohsaxton** [comment]
The same thing happens to me on an M1 iMac running OS 13.3.1, but not every time I start Frescobaldi.  On average it seems to happen about 2/3 of the time.

**Comment by lachi1905** [comment]
Same thing on M1 MacBook Air, OS 13.6. Uninstall and then reinstall with macports didn't help. After restarting the MacBook  Frescobaldi works fine until quitting it, then trying to restart leads to the described problem. In my case it seems that waiting a minute before restarting Frescobaldi helps: it worked fine the last 5 times I waited, then a quick start showed the inactive window and again after force quit and waiting a minute it worked fine...

**Comment by bishopcranmer** [comment]
Not to pile on — and I see that this is nearly a year old — but this is a continuous problem for me as well under Sonoma (14.4.1).

**Comment by inkandpaper-app** [comment]
I noticed that the problem never occurs when launching the app executable 'Frescobaldi' from the terminal. Instead, the app very often becomes unresponsive if it is launched with a double click on the icon. I hope this can help.

**Comment by lachi1905** [comment]
Yes, it does help, thanks 
(14.6.1)

**Comment by bishopcranmer** [comment]
> I noticed that the problem never occurs when launching the app executable 'Frescobaldi' from the terminal. Instead, the app very often becomes unresponsive if it is launched with a double click on the icon. I hope this can help.

This is an excellent pointer and works perfectly. Worth a few extra steps. For those who aren't sure how to make this work, do the following.

1. Find the file Frescobaldi.app.
2. Right-click (control-click) on the app and select "Package Contents"
3. Under "Contents", go to "MacOS".
4. Open "Terminal" and drag the "Frescobaldi" file in "Contents --> MacOS" onto the "Terminal" window.
5. Press "Enter/Return".

Voila!

**Comment by inkandpaper-app** [comment]
Simple hack that seems to work (can anyone else test this?)

1) Rename the file 
/Applications/Frescobaldi.app/Contents/MacOS/Frescobaldi 
to
/Applications/Frescobaldi.app/Contents/MacOS/F

2) Create and save in the folder /Applications/Frescobaldi.app/Contents/MacOS/
a simple bash file called 'Frescobaldi' with the two lines:

#!/bin/bash
exec "$(dirname "$0")/F"

3) Make this file executable:
chmod +x /Applications/Frescobaldi.app/Contents/MacOS/Frescobaldi

**Comment by lachi1905** [comment]
> Simple hack that seems to work (can anyone else test this?)

Didin't work on my M1 MacBook Air with 14.6.1, had the same issues as usual. Still working well when launching it from Terminal

**Comment by jeanas** [comment]
I'm pretty sure this is a duplicate of #428.

**Comment by lachi1905** [comment]
> > I noticed that the problem never occurs when launching the app executable 'Frescobaldi' from the terminal. Instead, the app very often becomes unresponsive if it is launched with a double click on the icon. I hope this can help.
> 
> This is an excellent pointer and works perfectly. Worth a few extra steps. For those who aren't sure how to make this work, do the following.
> 
> 1. Find the file Frescobaldi.app.
> 2. Right-click (control-click) on the app and select "Package Contents"
> 3. Under "Contents", go to "MacOS".
> 4. Open "Terminal" and drag the "Frescobaldi" file in "Contents --> MacOS" onto the "Terminal" window.
> 5. Press "Enter/Return".
> 
> Voila!

You can also drag the Frescobaldi file to the Dock, so you need this procedure only once and can then open the app with just one click
